### ISSUE TITLE
For Cycle 4

### ISSUE THREAD

**Issue by HarryMacfinned** [bisq-network/compensation#330]
## Summary

> _Specify the total amount of BSQ you are requesting_

 - BSQ requested: <del>`2900`</del> `1600`

This compensation request :
- follows the https://docs.bisq.network/roles.html#compensation recommandations
- applies the 1BSQ=1USD rule (as ~everybody does)

### Substantive reports

- Support | https://github.com/bisq-network/roles/issues/64#issuecomment-519627588 | (BSQ requested = <del>`2500`</del> `1200`)
<del>Evaluation is based and intended in line with : previous compensation requests and other support compensation requests</del>.

- Fees and connectivity monitoring | see the weekly reports in https://github.com/bisq-network/roles/issues/10 | (BSQ requested = `200`)
Concerned monitored data files : https://github.com/HarryMacfinned/bisq-monitor
TL;DR: NTR, nodes were delivering coherent fees.

- Prices monitoring | https://github.com/bisq-network/roles/issues/10#issuecomment-519581596 | (BSQ requested = `200`)
TL;DR: 0 alerts transmitted this cycle.

NB1 : The monitoring role includes also manual sorting of false alarms, ie monitoring several Bisq specific pages + following of Tor metrics page + maillist
NB2 : All the alerts are given on the slack #bisq-monitor and #bisq-pricenode channels

## Others
- Clearnet Bisq webpages monitoring : reported in the #bisq-website channel [`0 BSQ`]
- PR https://github.com/bisq-network/bisq/pull/2947 listing delisted assets, merged [`0 BSQ`]
- PR https://github.com/bisq-network/bisq-docs/pull/144 typos + addition for howto/list-asset.adoc [`0 BSQ`]
- PR https://github.com/bisq-network/bisq-markets/pull/18 3 files updated, waiting for review/merge

**Comment by HarryMacfinned** [comment]
I did notice that my above compensation request wasn't voted.

Since I didn't get any comment up to know, I can only make some guesses.
My main guess is that the blackballing concerns the support role and the way I handle it recently.
I have to say (and I already did it in my previous reports) that I find myself that there are some worries with the support (as a role), mainly given that ~90% of the support is used for questions concerning < 5% of Bisq's business.
In this regard, expenses for support role can legitimately be considered as expensive.

I don't think that Bisq's business will change drastically in the next months (all the clues says so).
And so, the support role as I handle it atm will probably still be questionable for the next cycles.
For this reason, I think that it's better that I stop with support as a (full attention) role.
(This will also let me more time to potentially use on other issues).
I will however still continue with punctual/spontaneous support when I can. 

I'll not close immediately the present issue, in case some contributors would be kind enough 
to write comments or suggestions (very welcomed).
Please, do not hesitate to give your opinion.

**Comment by MwithM** [comment]
Closed as rejected.

**Comment by MwithM** [comment]
Accepted on Cycle 5.

**Comment by MwithM** [comment]
Closed.
